### PR TITLE
https://issues.redhat.com/browse/ACM-4404 : Updating links

### PR DESCRIPTION
**Pull request by faaden123**

### CODE REVIEW

File: applications/gitops_config.adoc [hunk 1]
```diff
@@ -14,7 +14,7 @@ To configure GitOps, you can register a set of one or more {product-title} manag
 [#register-gitops]
 == Registering managed clusters to GitOps
 
-. Create managed cluster sets and add managed clusters to those managed cluster sets. See the example for managed cluster sets in the https://github.com/open-cluster-management/multicloud-integrations/blob/main/examples/managedclusterset.yaml[multicloud-integrations managedclusterset].
+. Create managed cluster sets and add managed clusters to those managed cluster sets. See the example for managed cluster sets in the https://github.com/stolostron/multicloud-integrations/blob/main/examples/openshift-gitops/managedclusterset.yaml[multicloud-integrations managedclusterset].
```
Comment (swopebe) [inline applications/gitops_config.adoc:17]: we are getting dinged on pointing to GH repos but we will discuss later. Looks good.

Comment (swopebe) [reply applications/gitops_config.adoc:17]: Let's take a moment here and use the link checker. Download "Check my links" from Chrome (if you have not already) and go to "preview" in this PR, then run it. There are a couple broken links in other spots. This is good to do just to mitigate broken links at GA; I can show you tomorrow.

Comment (faaden123) [reply applications/gitops_config.adoc:17]: Just ran it. It shows me the one broken link, but indicates there is three. Extremely helpful, thank you.

Comment (faaden123) [reply applications/gitops_config.adoc:17]: Found a link that isn't 1:1 but does provide useful on placement information for ManagedClusters-and leads users to further info on ManagedClusters vs ManagedClusterSets.

Let me know what you think

File: applications/gitops_config.adoc [hunk 2]
```diff
@@ -30,7 +30,7 @@ See the link:../clusters/cluster_lifecycle/create_clustersetbinding.adoc#creatin
 . In the namespace that is used in managed cluster set binding, create a placement custom resource to select a set of managed clusters to register to an {ocp-short} GitOps operator instance. You can use the example in the repository at https://github.com/stolostron/multicloud-integrations/blob/main/examples/placement.yaml[multicloud-integration placement]
 
 +
-See link:../clusters/cluster_lifecycle/placement_managed.adoc[Using ManagedClusterSets with Placement] for placement information. 
+See link:../clusters/cluster_lifecycle/placement_intro.adoc[Selecting ManagedClusters with Placement] for placement information.
```
Comment (swopebe) [inline applications/gitops_config.adoc]: I think this should work.

**Review by swopebe — APPROVED** [review]
Be sure to get @ch-stark to approve.

**Comment by swopebe** [comment]
@dockerymick thanks for offering to help on this one.

**Pull request merged**